locksmith/src/operations/wedlocksOperations.ts

```diff
@@ -68,6 +70,16 @@ export const sendEmail = async (
     attachments,
   }
 
+  // prevent send email when is not enabled
+  const canSendEmail = await isSendEmailEnabled(
+    params.lockAddress,
+    Number(params.network)
+  )
+
+  if (!canSendEmail) {
```

Comment:
```suggestion
  const canSendEmail = await isSendEmailEnabled(
    params.lockAddress,
    Number(params.network)
  )

  if (!canSendEmail) {
```

The function now is inside `sendEmail`, to improve next we can refactor this part in the future in order to first check if we can send an email and if so, run the rest of the code.